### PR TITLE
Database.go put that in your pipe and smoke it

### PR DESCRIPTION
**Pull request by TylerMetz**
## Describe your changes
Database.go is almost there

## Issue ticket number and link
#430

### CODE REVIEW

**Comment by TylerMetz** [comment]
I think the list page is breaking this? I merged develop into it and then frontend wouldnt run but the backend works? If we merge would it be fine?

File: Saucier720-app/src/app/core/interfaces/ingredient.ts
Comment (samforstot) [inline Saucier720-app/src/app/core/interfaces/ingredient.ts:1]: W

**Review by samforstot — APPROVED** [review]
LGTM. Really appreciate the work you put into this. Transition to Azure is so hot

**Review by rileycleavenger — CHANGES_REQUESTED** [review]
LGTM! but fix before merge to develop

**Comment by TylerMetz** [comment]
We had to revert to before the merge conflicts. I did them bad

**Pull request closed without merging**